test/timecop_with_process_clock_test.rb

```diff
@@ -0,0 +1,168 @@
+require_relative "test_helper"
+require 'timecop'
+
+class TestTimecopWithProcessClock < Minitest::Test
+  TIME_EPSILON = 0.001 # seconds - represents enough time for Process.clock_gettime to have advanced if not frozen
```

Comment:
Open to other ideas for this constant name to better represent the meaning that I'm currently explaining with the comment.